src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowVscTest.java

```diff
@@ -450,6 +481,7 @@ void testAcEmuWithOperationalLimits() {
         Network network = HvdcNetworkFactory.createHvdcLinkedByTwoLinesAndSwitch(HvdcConverterStation.HvdcType.VSC);
         // without limit p=195
         network.getHvdcLine("hvdc23")
+                .setR(0d) //Removing resistance to ignore cable loss
```

Comment:
As a general comment, if unit tests changed because we improve/fix something, we prefer to fix them than to retry to code the old behavior.

Reply:
Ok, then the values of the tests have been updated with the new behavior (that have been verified). A new test had been added for the case without R to have an independant test ignoring line loss.